src/main/java/org/broadinstitute/consent/http/db/SamDAO.java

```diff
@@ -97,17 +97,13 @@ public UserStatus postRegistrationInfo(AuthUser authUser) throws Exception {
     HttpResponse response = executeRequest(request);
     String body = response.parseAsString();
     if (!response.isSuccessStatusCode()) {
-      if (HttpStatusCodes.STATUS_CODE_CONFLICT == response.getStatusCode()) {
-        throw new ConsentConflictException("User exists in Sam: " + authUser.getEmail());
-      } else {
-        String errorMsg = String.format("Error posting user registration information to Sam. Email [%s]. Status Code [%s]; Status Message [%s];  ",
+      String message = JsonParser.parseString(body).getAsJsonObject().get("message").getAsString();
+      String errorMsg = String.format("Error posting user registration information. Email: %s. Error message: %s.",
```

Comment:
removed "to Sam" because I don't think users need to know about Sam. I'm open to workshopping this message more. And I also want to improve the error message from Sam.

Reply:
I'm fine with this. This was helpful mostly just for me when looking at consent logs.